### PR TITLE
create rent model and references with model the user and book

### PR DESCRIPTION
**Pull request by MaaooVaass**
## Summary
create rent model and references with model the user and book

## Card
https://trello.com/c/Y9SsYf0j/20-crear-modelo-de-rent

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class User < ApplicationRecord
   devise :database_authenticatable, :registerable, :trackable
   include DeviseTokenAuth::Concerns::User
+  has_many :rents, dependent: :destry
```
Comment (lmasello) [inline app/models/user.rb]: `destroy`

Comment (lmasello) [inline app/models/user.rb]: `destroy`

Comment (lmasello) [reply app/models/user.rb]: Then the Book model should also have has_many rents. Do you agree?

**Review by lmasello — CHANGES_REQUESTED** [review]

File: app/models/user.rb (hunk 1 above)
Comment (MaaooVaass) [reply app/models/user.rb]: yes.

**Review by lmasello — APPROVED** [review]

**Review by fedeest — APPROVED** [review]

**Pull request merged**